app/assets/stylesheets/active_admin/mixins/_sections.scss

```diff
@@ -1,24 +1,16 @@
 @mixin section-header {
-  @include secondary-gradient;
-  @include text-shadow;
-  border: solid 1px #cdcdcd;
-  @include border-colors(#e6e6e6, #d4d4d4, #cdcdcd);
-  box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 0 1px #FFF inset;
-
+  background-color: #e9e9e9;
+  border-radius: 4px 4px 0 0;
   font-size: 1em;
   font-weight: bold;
-  line-height: 18px;
-  margin-bottom: 0.5em;
   color: $section-header-text-color;
   @include icon($section-header-text-color, 1.0em);
-
   padding: 5px 10px 3px 10px;
 }
 
 @mixin section-background {
-  background: #f4f4f4;
-  @include rounded(4px);
-  @include inset-shadow(0,1px,4px, #ddd);
+  background-color: #f8f8f8;
```

Comment:
Color literals like `#f8f8f8` should only be used in variable declarations; they should be referred to via variable everywhere else.